google/cloud/forseti/services/base/config.py

```diff
@@ -148,15 +149,50 @@ def get_storage_class(self):
 
 
 class InventoryConfig(AbstractInventoryConfig):
+    # pylint: disable=too-many-instance-attributes
     """Implements composed dependency injection for the inventory."""
 
+    @staticmethod
+    def _filter_valid_resources(excluded_resources):
+        """Filter valid excluded resources.
+
+        Args:
+            excluded_resources (list): A list of excluded resources to
+                validate against.
+
+        Returns:
+            list: List of valid resources to exclude.
+        """
+        valid_resource_patterns = [
```

Comment:
Rather than have the 'or' in the `__init__` below, have a check at the start of this function that returns an empty list of excluded_resources is False

```
if not excluded_resources
    return []
```